python/ray/tests/test_tracing.py

```diff
@@ -0,0 +1,46 @@
+import ray
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleExportSpanProcessor,
+)
+from typing import Any
+import pytest
+
+def _setup_tracing(*args: Any, **kwargs: Any) -> None:
+    if getattr(ray, "__traced__", False):
+        return
+
+    ray.__traced__ = True
+    # Sets the tracer_provider. This is only allowed once per execution context and will log a warning if attempted multiple times.
+    trace.set_tracer_provider(TracerProvider())
+    trace.get_tracer_provider().add_span_processor(
+        SimpleExportSpanProcessor(ConsoleSpanExporter())
+    )
+
+@pytest
+def test_tracing_task():
+    ray.worker.global_worker.run_function_on_all_workers(_setup_tracing)
+
+    ray.init()
+    @ray.remote
+    def f():
+        print("foo")
+
+    obj_ref = f.remote()
+    ray.get(obj_ref)
```

Comment:
The test doesn't seem to have verification of tracing? Is it still WIP?

Reply:
it's a wip. it will verify the traces via json file later.